src/dataframe/old/types.h

```diff
@@ -87,11 +92,10 @@ class Filter
 
 	[[nodiscard]] Filter operator&&(const Filter &other) const
 	{
-		auto first =
-		    func1.get() == True ? other.func1.get() : func1.get();
-		auto second =
-		    other.func1.get() == first ? True : other.func1.get();
-		return {first, second};
+		auto &&[min, max] =
+		    std::minmax({func1.get(), other.func1.get()});
```

Comment:
[unspecified](https://stackoverflow.com/questions/44397784/is-operator-less-than-on-pointers-consistent) by the compiler. It should use 2nd argument as `std::less<Fun*>` (and [not `less<>`](https://quuxplusone.github.io/blog/2019/01/20/std-less-nightmare/)).

Reply:
https://github.com/vizzuhq/vizzu-lib/pull/553/files#diff-af29f3d181986e3a0151b57f1598584001d2f9f67668313a4fb2f642e4011deaR96